pkgs/tools/security/iaito/default.nix

```diff
@@ -13,7 +13,7 @@
 
 let
   pname = "iaito";
-  version = "5.9.4";
+  version = "5.9.6";
```

Comment:
Looks like there's no source hash change for this derivation. Is this intentional?

Reply:
Ohh thanks for this. I actually used `nix-update` which seems to be not able to do the right thing with `iaito`.
I will try to work on it in a seperate PR while keep this PR scoped only for radare2.